### PR TITLE
Add adopters section in readme

### PR DESCRIPTION
**Pull request by jatcwang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,12 @@ You can find a list of libraries and integrations with data stores built on top
 
 If you have a project you'd like to include in this list, either open a PR or let us know in [the gitter channel](https://gitter.im/functional-streams-for-scala/fs2) and we'll add a link to it.
 
+### Adopters ### 
+
+Here's a (non-exhaustive) list of companies that use FS2 in production. Don't see yours? You can [add it in a PR!](https://github.com/functional-streams-for-scala/fs2/edit/main/README.md)
+
+*
```
Comment (Daenyth) [inline README.md:128]: Teikametrics

Not mine, but, Ovo?

Comment (kubukoz) [reply README.md:128]: @ocadotechnology, @cr-org

Comment (zoltanmagyar) [reply README.md:128]: Formedix

Comment (LukaJCB) [reply README.md:128]: CompStak

Comment (jatcwang) [reply README.md]: These are all added :)

**Comment by jatcwang** [comment]
Done. I think we can merge this now.

**Comment by kubukoz** [comment]
Thanks a lot! I think it's a good start.

**Comment by ChristopherDavenport** [comment]
Banno Group inside of Jack Henry & Associates

File: README.md [hunk 2]
```diff
@@ -121,6 +121,18 @@ You can find a list of libraries and integrations with data stores built on top
 
 If you have a project you'd like to include in this list, either open a PR or let us know in [the gitter channel](https://gitter.im/functional-streams-for-scala/fs2) and we'll add a link to it.
 
+### Adopters ### 
+
+Here's a (non-exhaustive) list of companies that use FS2 in production. Don't see yours? You can [add it in a PR!](https://github.com/functional-streams-for-scala/fs2/edit/main/README.md)
+
+* [Chatroutelette](https://v2.chatroulette.com/)
```
Comment (mpilquist) [inline README.md:128]: Typo

**Comment by mpilquist** [comment]
Looking good. Let’s add Comcast too.

File: README.md (hunk 2 above)
Comment (jatcwang) [reply README.md:128]: whoops fixed.

Comment (jatcwang) [reply README.md]: @mpilquist ready to merge when you are :)

**Pull request merged**